### PR TITLE
ambient: comment no gateway as default

### PR DESCRIPTION
**Pull request by daixiang0**
## Description

Fix https://github.com/istio/istio/issues/50219

## Reviewers

- [x] Ambient
- [x] Docs
- [x] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [x] User Experience
- [ ] Developer Infrastructure
- [ ] Localization/Translation

### CODE REVIEW

File: content/en/docs/ops/ambient/getting-started/index.md [hunk 1]
```diff
@@ -101,6 +101,11 @@ four components (including {{< gloss "ztunnel" >}}ztunnel{{< /gloss >}}) have be
 
 {{< /tabset >}}
 
+{{< tip >}}
+Note that ambient mode does not include ingress gateway as default.
+{{< /tip >}}
+
+
```
Comment (wilsonwu) [inline content/en/docs/ops/ambient/getting-started/index.md:108]:
```suggestion
{{< tip >}}
Note that ambient mode does not include ingress gateway as default.
{{< /tip >}}

```

**Comment by ericvn** [comment]
I'm not sure that we need this called out here. We already document that there is in ingress gateway or egress gateway for the ambient profile on the profile page: https://istio.io/latest/docs/setup/additional-setup/config-profiles/

**Comment by dhawton** [comment]
> I'm not sure that we need this called out here. We already document that there is in ingress gateway or egress gateway for the ambient profile on the profile page: https://istio.io/latest/docs/setup/additional-setup/config-profiles/

+1 we don't need to call it out here. We don't call out in our demos when installing the `default` profile that it does not have ingress gateway. Referencing the config-profiles doc is the best action and reduces information duplication.

**Comment by frankbu** [comment]
> +1 we don't need to call it out here. We don't call out in our demos when installing the default profile that it does not have ingress gateway. Referencing the config-profiles doc is the best action and reduces information duplication.

The default profile DOES have ingress gateway.

I think it's helpful if the tip is changed as I suggested above (https://github.com/istio/istio.io/pull/14884#discussion_r1567360541), it's just a tip to point out that the ingress gateway needs to be explicitly enabled if using the bookinfo services in ambient.

That said, I'm also OK with not highlighting anything about installing gateways along with istiod, since we now recommend gateways be installed separately.

**Comment by daixiang0** [comment]
@frankbu thank you, I have updated.

**Review by frankbu — APPROVED** [review]

**Pull request merged**